chessgame/src/common/pieces/Pawn.ts

```diff
@@ -0,0 +1,32 @@
+import Piece from "./_Interface";
+
+class Pawn extends Piece {
+    type: string = "Pawn";
+
+    canMove(column: string, row: number): boolean {
```

Comment:
Ogolnie to moze na ukos tez sie ruszac i wiele innych. Tu jest spoko wyjasnione wszystko: https://www.chess.com/terms/chess-pawn. Ale mysle, ze poki cos tak zostawimy i potem sie zrobi ze ta canMove metoda gameState tez przyjmuje i w zaleznosci od tego sie bedzie jeszcze ruchy walidowac.